src/main/java/com/github/princesslana/eriscasper/util/FormatType.java

```diff
@@ -0,0 +1,32 @@
+package com.github.princesslana.eriscasper.util;
+
+import com.github.princesslana.eriscasper.ErisCasperFatalException;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.io.OutputStream;
+import javax.imageio.ImageIO;
+
+public enum FormatType {
+  GIF("gif"),
+  JPG("jpg"),
+  JPEG("jpeg"),
```

Comment:
I don't think we need JPG and JPEG.

Reply:
alright, they are different formats according to ImageIO which is why I added it